src/ray/gcs/gcs_client/test/global_state_accessor_test.cc

```diff
@@ -51,14 +76,19 @@ class GlobalStateAccessorTest : public ::testing::Test {
       std::this_thread::sleep_for(std::chrono::milliseconds(10));
     }
 
-    // Create GCS client.
-    gcs::GcsClientOptions options(config.redis_address, config.redis_port,
-                                  config.redis_password);
-    gcs_client_ = std::make_unique<gcs::GcsClient>(options);
+    // Create GCS client and global state.
+    if (RayConfig::instance().bootstrap_with_gcs()) {
+      gcs::GcsClientOptions options("127.0.0.1:6379");
```

Comment:
Why not generate the address `127.0.0.1:6379` from config?

Reply:
I did this because there is no gcs_ip in the config. The best thing we can do probably is like: "127.0.0.1" + config.gcs_port.